.github/workflows/github-action.yml

```diff
@@ -1,6 +1,7 @@
 name: Elixir CI
 
 on:
+  workflow_dispatch:
```

Comment:
hmm 🤔 
we don't want this for all jobs.

we need a new job that will help us deploy to gigalixir which should be a manual trigger. 
The test and docker builds should happen automatically